sections/introduction.include

```diff
@@ -627,23 +633,23 @@
 
   In the following example, the page has been made yellow-on-blue using CSS.
 
-  <xmp highlight="html">
-    <!DOCTYPE html>
-    <html>
-      <head>
-        <title>Sample styled page</title>
-        <style>
-          body {
-            background: blue;
-            color: yellow;
-          }
-        </style>
-      </head>
-      <body>
-        <h1>Sample styled page</h1>
-        <p>This page is just a demo.</p>
-      </body>
-    </html>
+  <xmp id="intro-early-example" highlight="html">
+  <!DOCTYPE html>
+  <html>
+    <head>
+      <title>Document title</title>
+      <style>
+        body {
+          background: blue;
+          color: yellow;
+        }
+      </style>
+    </head>
+    <body>
+      <h1>Styled page</h1>
+      <p>This is a paragraph has yellow text and the body element has a blue background.</p>
```

Comment:
Appears the paragraph is missing a word 
>"This is a paragraph **_which_** has yellow..." 

If we're going to change the text to explicitly call out the visual styling, it should probably say something like:
>"This document has yellow text and a blue background"

It's not just the `p`, but the `h1` which would have the yellow text.

Reply:
Thanks for the feedback @scottaohara. I've just pushed an update for both of these comments.